milli/src/search/new/ranking_rule_graph/position/mod.rs

```diff
@@ -105,8 +106,20 @@ impl RankingRuleGraphTrait for PositionGraph {
             ));
         }
 
+        // artificial empty condition for computing max cost
+        let max_cost = term.term_ids.len() as u32 * 10;
+        edges.push((
+            max_cost,
+            conditions_interner
+                .insert(PositionCondition { term: term.clone(), positions: Vec::default() }),
+        ));
```

Comment:
What happens if an edge already exists with this cost? Generally, I tried to have only one condition associated with each cost. I think everything still works correctly if that's not the case, but still, it seems conceptually simpler to have 1 cost -> 1 condition.

Reply:
Ah, I assumed that since it could be the case for edges created by distinct calls to `build_edges` to have the same cost, it was also OK inside of a single call.

Do you want me to add a special kind of condition then? Or to check that the max cost wasn't already added in the look above?

Reply:
It should work correctly either way, but it does make it easier to debug when no duplicate costs exist :)